### DISCUSSION TITLE
Connection with Regatta app?

### DISCUSSION THREAD

**Discussion by MikePierce-Regatta** [JaapvanEkris/openrowingmonitor#19] [Ideas]
Hey there - love this concept. Can you test this with the app Regatta for compatibility? It recognizes both Concept2 BLE and FTMS protocols. Thanks!
https://play.google.com/store/apps/details?id=com.teamregatta&hl=en_US&gl=US

**Comment by JaapvanEkris** [comment]
No problem! I'll try them this weekend!

**Reply by JaapvanEkris** [reply]
I did a first test on the simulator (we can simulate rows based on recorded tracks), and both C2 and FTMS work. I need some more testing to see how it behaves long-term, but first indications are pretty good.

**Comment by MikePierce-Regatta** [comment]
Nice and thanks for testing!  Which machine are you using? On May 25, 2023, at 7:19 AM, Jaap van Ekris ***@***.***> wrote:﻿
I did a first test on the simulator (we can simulate rows based on recorded tracks), and both C2 and FTMS work. I need some more testing to see how it behaves long-term, but first indications are pretty good.

**Reply by JaapvanEkris** [reply]
> Nice and thanks for testing!  Which machine are you using?

I test on a Concept2 RowErg, as I am also validating the ORM Physics Engine. But I've rowed on a NordicTrack and similar machines as well. For an overview of supported machines, see https://github.com/JaapvanEkris/openrowingmonitor/blob/V1Beta_Architecture_Revision/docs/Supported_Rowers.md
